ui.frontend/src/pages/ConsolePage.tsx

```diff
@@ -33,11 +33,7 @@ const ConsolePage = () => {
   const [code, setCode] = useState<string | undefined>(localStorage.getItem(StorageKeys.EDITOR_CODE) || ConsoleCode);
   const [autoscroll, setAutoscroll] = useState<boolean>(true);
   const [execution, setExecution] = useState<Execution | null>(null);
-  const [compiling, syntaxError, compileError, parseExecution ] = useCompilation(code);
```

Comment:
why are you removing it?

I intentionally added it because without full errors are not visible after clicking 'see errors' in compilationstatus component

Reply:
We have to discuss then, If there is stored execution it will clashes with execution which is basically just parsing

Reply:
no it should not clash as parsing is done before execution; different timing